spec/features/update_case_information_from_allocation_page_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.feature "Update case information from allocation page", type: :feature do
+  let(:offender) { build(:nomis_offender, complexityLevel: 'high', agencyId: prison.code) }
+  let(:offenders) { [offender] }
+  let(:pom) { build(:pom) }
+  let(:spo) { build(:pom) }
+  let(:prison) { create(:prison) }
+  let(:offender_no) { offender.fetch(:offenderNo) }
+
+  before do
+    stub_offenders_for_prison(prison.code, offenders)
+    stub_signin_spo(spo, [prison.code])
+    stub_poms(prison.code, [pom, spo])
+    stub_keyworker(prison.code, offender.fetch(:offenderNo), build(:keyworker))
+  end
+
+  context 'when updating the case information' do
```

Comment:
It looks like this top level context block isn't really needed, since it wraps all the tests in this spec.